lib/index.js

```diff
@@ -36,10 +39,63 @@ const tmpTsconfig = {
     skipLibCheck: true,
   },
   files,
-  include: [],
+  include: typeRoots,
 }
 fs.writeFileSync(tmpTsconfigPath, JSON.stringify(tmpTsconfig, null, 2))
 
+/*
+  The process might be terminated before the temporary config cleanup is done.
+  To prevent that we'll install exit listeners on termination signals to ensure
+  that the file is cleaned up properly.
+*/
+let didCleanup = false
+const exitCallback = function (
+  signal,
+  /*
+    As documented in https://nodejs.org/api/process.html#signal-events, setting
+    up listeners for *some* signals makes Node.js disable automatic process exit
+    when they are emitted, thus shouldExit is necessary
+  */
+  shouldExit
+) {
+  return function (exitCode) {
+    if (didCleanup) {
+      return
+    }
+    didCleanup = true
+    try {
+      fs.unlinkSync(tmpTsconfigPath)
+    } catch (error) {
+      console.error(`Failed to clean up file during signal ${signal}`, error)
+    }
+    if (shouldExit) {
+      process.exit(exitCode)
+    }
+  }
+}
+
+// Handle termination events from https://nodejs.org/api/process.html#signal-events
+for (const signal of [
+  'exit',
+  'SIGINT',
+  'SIGTERM',
+  'SIGQUIT',
+  'SIGHUP',
+  'SIGPIPE',
+]) {
+  process.on(
+    signal,
+    exitCallback(
+      signal,
+      /*
+        The "exit" signal is emitted when Node.js is exiting normally, thus
+        (shouldExit = true) is not necessary in that case
+      */
+      signal === 'exit' ? false : true
```

Comment:
This logic can be simplified. You should rarely return an `boolean` from a `boolean` expression when you can return the value of the `boolean` expression.

```suggestion
      signal !== 'exit'
```